docs/devices/7146060PH.md

```diff
@@ -59,6 +59,9 @@ This may also be possible with the
 by pressing and holding the reset button on the bottom of the remote (next to the battery).
 [This may not always work, even if the Hue bulb starts flashing](https://github.com/Koenkk/zigbee2mqtt/issues/296#issuecomment-416923751).
 
+#### Button long-press
+The first-generation Hue Go can be reset by long pressing the button on the bottom of the unit, [as advised by Philips on Twitter](https://twitter.com/tweethue/status/937713840728559621).
+They suggest keeping the button pressed for 35 seconds. After about 25 seconds the light will start doing some flashes: keep presseing anyway for the whole 35.
```

Comment:
I agree, done! Furthermore, that comment was hardly ever useful, because it refers to a video about the dimmer switch while giving a solution that doesn't need it.